src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public enum Color {
+    Red,
```

Comment:
```suggestion
    RED,
```
enum values should be in uppercase